Tests/Cell.py

```diff
@@ -977,6 +977,45 @@ def test_execute_mdx_csv(self):
             self.total_value,
             sum(values))
 
+    def test_execute_mdx_csv_row_only(self):
```

Comment:
should be `test_execute_mdx_csv_column_only`

Reply:
Oh yeah! got confused while fixing it.